mutatex/core.py

```diff
@@ -759,6 +759,7 @@ def __init__(self,
                 output_processing={},
                 link_files=False,
                 write_log=False,
+                values_check={},
```

Comment:
I understand the purpose of this - and I think ti works perfectly, but can we change it's name? It's not much for checking "values" but for checking if there were problems even though foldx exited with status 0. It could be something like `output_check` (similar to `output_processing`)